### ISSUE TITLE
Project templates

### ISSUE THREAD

**Issue by mayakwd** [HaxeFoundation/intellij-haxe#819]
We need to lower the entry threshold for developers by supporting predefined project templates.
- [ ] OpenFl/Lime
- [ ] Kha
- [ ] Flixel
- [ ] Heaps
- [ ] ?

**Comment by EricBishton** [comment]
I agree wholeheartedly. Then maybe Nicolas will change his mind about pushing VSHaxe as the standard point of entry. 😀
